consensus/safety-rules/src/serializer.rs

```diff
@@ -33,19 +45,31 @@ impl SerializerService {
     pub fn handle_message(&mut self, input_message: Vec<u8>) -> Result<Vec<u8>, Error> {
         let input = bcs::from_bytes(&input_message)?;
 
-        let output = match input {
-            SafetyRulesInput::ConsensusState => bcs::to_bytes(&self.internal.consensus_state()),
-            SafetyRulesInput::Initialize(li) => bcs::to_bytes(&self.internal.initialize(&li)),
-            SafetyRulesInput::ConstructAndSignVote(vote_proposal) => {
-                bcs::to_bytes(&self.internal.construct_and_sign_vote(&vote_proposal))
-            }
-            SafetyRulesInput::SignProposal(block_data) => {
-                bcs::to_bytes(&self.internal.sign_proposal(*block_data))
-            }
-            SafetyRulesInput::SignTimeout(timeout) => {
-                bcs::to_bytes(&self.internal.sign_timeout(&timeout))
-            }
-        };
+        let output =
+            match input {
+                SafetyRulesInput::ConsensusState => bcs::to_bytes(&self.internal.consensus_state()),
+                SafetyRulesInput::Initialize(li) => bcs::to_bytes(&self.internal.initialize(&li)),
+                SafetyRulesInput::ConstructAndSignVote(vote_proposal) => {
+                    bcs::to_bytes(&self.internal.construct_and_sign_vote(&vote_proposal))
+                }
+                SafetyRulesInput::SignProposal(block_data) => {
+                    bcs::to_bytes(&self.internal.sign_proposal(*block_data))
+                }
+                SafetyRulesInput::SignTimeout(timeout) => {
+                    bcs::to_bytes(&self.internal.sign_timeout(&timeout))
+                }
+                SafetyRulesInput::SignTimeoutWithQC(timeout, maybe_tc) => bcs::to_bytes(
```

Comment:
Maybe add a comment about the below messages only for two-chain?

Reply:
the comment is on the public api (TSafetyRules)